test/presenters/worldwide_corporate_information_page_presenter_test.rb

```diff
@@ -42,7 +42,7 @@ def schema_name
 
     presented = create_presenter(WorldwideCorporateInformationPagePresenter, content_item: without_sponsoring_organisations)
 
-    expected = { name: "British Embassy Manila", url: "/world/organisations/british-embassy-manila", crest: "single-identity", brand: "single-identity" }
+    expected = { name: "British Embassy<br/>Manila", url: "/world/organisations/british-embassy-manila", crest: "single-identity", brand: "single-identity" }
```

Comment:
I don't understand where this <br/> is coming from. I can see the [title](https://github.com/alphagov/publishing-api/blob/main/content_schemas/examples/worldwide_corporate_information_page/frontend/worldwide_corporate_information_page.json#L41) in the schema example...

Reply:
It comes from the [`formatted_title`](https://github.com/alphagov/publishing-api/blob/56d487e606b280551aa0f94942b21d18c2cf2416/content_schemas/examples/worldwide_corporate_information_page/frontend/worldwide_corporate_information_page.json#L106).

Reply:
Oh different field! OK gottit!